.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": "No Configurations"
```

Comment:
What are the .vs files for?

Reply:
Pretty sure the whole `.vs` folder can be ignored.